### PR TITLE
Handle downcased name in emacs modeline

### PR DESCRIPTION
**Pull request by masatake**

### CODE REVIEW

File: main/parse.c [hunk 1]
```diff
@@ -2567,9 +2568,9 @@ static boolean makeKindDescriptionPseudoTag (kindOption *kind,
 	vStringPut (letter_and_name, ',');
 	vStringCatS (letter_and_name, kind -> name);
 
+	d = kind -> description? kind -> description: kind -> name;
```
Comment (b4n) [inline main/parse.c]: personal style opinion:  I'd find this more readable as
`d = kind->description ? kind->description : kind->name;`

**Comment by b4n** [comment]
Diff LGTM, but I'm confused but the PR title

**Comment by masatake** [comment]
> , but I'm confused but the PR title

It looks something wrong in github side...

**Comment by masatake** [comment]
I withdraw this PR. I will make a new PR after updating the sytle and title.

**Pull request closed without merging**